Installer/make_release_packages.bat

```diff
@@ -16,9 +16,8 @@ pause
 REM git clone -b v2.8RC1 https://github.com/asterics/AsTeRICS
 REM clone Asterics including submodules and submodules of submodules
 REM git clone --recurse-submodules https://github.com/asterics/AsTeRICS.git
-git clone -b Branch_Bugfixes_v3.0.1 --recurse-submodules https://github.com/asterics/AsTeRICS.git
+git clone -b Branch_Bugfixes_v3.0.1
```

Comment:
here now somehow the repository URL is missing...

Reply:
do we still need this script `make_release_packages.bat`? or could it be removed and only `jenkins-release-script.sh` be used?

Reply:
uuups, sorry, removed too much.
Yes I would prefer to only keep the .sh script as it runs on windows 10 anyway because of the Linux support.
I will remove the .bat file